examples/common/python/connectors/direct/tcs_listener/tcs_listener.py

```diff
@@ -74,8 +77,18 @@ def __init__(self, config):
         self.worker_registry_handler = TCSWorkerRegistryHandler(self.kv_helper)
         self.workorder_handler = TCSWorkOrderHandler(
             self.kv_helper, config["Listener"]["max_work_order_count"])
-        self.workorder_receipt_handler = TCSWorkOrderReceiptHandler(self.kv_helper)
-        self.worker_encryption_key_handler = WorkerEncryptionKeyHandler(self.kv_helper)
+        self.workorder_receipt_handler = TCSWorkOrderReceiptHandler(
+            self.kv_helper)
+        self.worker_encryption_key_handler = WorkerEncryptionKeyHandler(
+            self.kv_helper)
+
+        self.dispatcher = Dispatcher()
+        rpc_methods = [
+            self.workorder_handler.WorkOrderSubmit,
+            self.workorder_handler.WorkOrderGetResult,
+        ]
+        for m in rpc_methods:
+            self.dispatcher.add_method(m)
```

Comment:
So in subsequent PRs we have a mapping between jrpc method and function for all requests ?

Reply:
Yes~